clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2122,6 +2122,8 @@ def GlobalOp : CIR_Op<"global",
                        OptionalAttr<TLSModel>:$tls_model,
                        // Note this can also be a FlatSymbolRefAttr
                        OptionalAttr<AnyAttr>:$initial_value,
+                       // whether op has comdat, its selection kind is ANY
```

Comment:
Does this comment needs update? either remove it or say "equivalent to LLVM's comdat any" or something along those lines